packages/plugin-renderer-react/package.json

```diff
@@ -31,6 +31,9 @@
     "isomorphic-fetch": "^2.2.1",
     "prop-types": "^15.5.8",
     "react-hot-loader": "^3.0.0-beta.7",
+    "react-router-dom": "^4.2.2",
```

Comment:
This is not an option for us. We can't use RR v4, see https://github.com/phenomic/phenomic/issues/1130

Reply:
Well, finally someone came up with a solution not on phenomic side. you can find it here https://github.com/parisjs/parisjs-website/commit/a044ffb5250bc6f897df7fe4a8fdb54527315ed7

It consists of having a `hashLinkScroll` function:
```js
function hashLinkScroll() {
   const { hash } = window.location
 
   if (hash !== '') {
     setTimeout(() => {
       const id = hash.replace('#', '')
       const element = document.getElementById(id)
       if (element) element.scrollIntoView()
     }, 0)
   }
 }
```

Then, you'd use it like so `<Router ... onUpdate={ hashLinkScroll } > ...</Router>`.
So maybe, we just need to document this or find a way to automatically inject the `hashLinkScroll` in phenomic once and for all. What are your thoughts on this?